.bazelrc

```diff
@@ -1,9 +1,6 @@
 # Configurations for arguments that should automatically be added to Bazel commands.
 build --android_databinding_use_v3_4_args \
```

Comment:
Per discussion with RD, update this file to include setting java_runtime_version to remotejdk_11 (or whatever the default is for builds; we should keep them the same).

Also, update docs since installing JDK should now be simpler.

Reply:
Addressed in latest changes.